challenge/src/App.jsx

```diff
@@ -1,45 +1,270 @@
 import React, { Component } from "react";
 import BabyYoda from "./mandalorian/BabyYoda";
-import NumberLogger from './NumberLogger';
+import NumberLogger from "./NumberLogger";
 import strings from "./strings.json";
-import Navbar from './components/Navbar'
-import Fade from 'react-reveal/Fade'
+import Navbar from "./components/Navbar";
+import Fade from "react-reveal/Fade";
 
 class App extends Component {
-    state = {}
-    constructor(props) {
-        super(props);
-    }
-    render() {
-        console.log("Render Method is Working. Redered successfully! I think?")
+  state = {};
```

Comment:
just a note, please avoid reformatting or reindenting entire files in a pr for some issue, it makes it hard to see where actual changes have been made. instead they should be submitted in a different pr. (and a realy project should be set up with a linter to vet code, but this isn't a real project XD)

Reply:
Sorry about that! I forgot to disable my code formatter before beginning, I will do so from now on.